draft-sharabayko-mops-srt.md

```diff
@@ -2862,13 +2862,21 @@ IV = (MSB(112, Salt) << 2) XOR (PktSeqNo)
 
 # Security Considerations
 
-SRT supports confidentiality of user data using stream ciphering based on AES.
-Session keys for ciphering are delivered through control packets during handshake,
-with the protection by Key Encryption Key, which is generated by a sender and receiver
-with pre-shared secret such as passphrase. As in UDT, careful uses of SYN Cookies
-may help to deter denial of service attacks. Appropriate security policy including
-key size, key refresh period, as well as passphrase should be managed by security
-officers, which is out of scope of the present document.
+SRT provides confidentiality of user data using stream ciphers as
+specified in {{encryption}}, which is the only improvement of
+security upon UDT. That means other limitations from UDT remain
+as they are, and SRT needs to consider any security protection
+mechanisms at an upper layer depending on its security requirements.
+Integrity protection of user data can also be supported as described
+in counter mode of {{SP800-38A}}, and its usage details are out of scope for the present document.
+
+The encryption feature of SRT requires some cautions. SEK needs to be changed with an
+appropriate refresh period to avoid any issues which might occur if security keys
+would be used for a long period of time. The shared secret for KEK generation needs
+to be carefully configured by a security officer, who is supposed to set security policies
+to force encryption and limit key size selection. Another aspect to consider is caching of KMmsg.
+When encryption and decryption of the same messages affect processing cost for sender or receiver,
+cached KMmsg could be retransmitted or checked before deciphering.
```

Comment:
There are some important considerations regarding the encryption feature of SRT:

  - The SEK must be changed at an appropriate refresh interval to avoid the risk 
  associated with the use of security keys over a long period of time. 
  - The shared secret for KEK generation must be carefully configured by a 
  security officer responsible for security policies, enforcing encryption, and 
  limiting key size selection. 
  - To improve performance, in some cases the KMmsg cache can be retrasnmitted 
  or checked before deciphering.

Reply:
Given some cautions are mentioned, do we need to mention here that SHA-1 hashing function is used?

Reply:
This one is not clear to me in the part of what is the performance issue and how caching of KMmsg solves it.

> To improve performance, in some cases the KMmsg cache can be retransmitted
or checked before deciphering.

Reply:
See the original text by Joonwoong. He meant that it can be cashed and not encrypted several times. But I don't know whether it makes sense.

Reply:
I don't think it is something relevant, because the same KMmsg is not expected to be sent periodically.
I suggest removing this item then.